mne/channels/montage.py

```diff
@@ -505,23 +505,40 @@ def read_dig_dat(fname):
         items = line.split()
         if not items:
             continue
-        elif len(items) != 5:
+        elif not (len(items) == 4 or len(items) == 5):
             raise ValueError(
                 "Error reading %s, line %s has unexpected number of entries:\n"
                 "%s" % (fname, i, line.rstrip()))
-        num = items[1]
-        if num == '67':
-            continue  # centroid
-        pos = np.array([float(item) for item in items[2:]])
-        if num == '78':
-            nasion = pos
-        elif num == '76':
-            lpa = pos
-        elif num == '82':
-            rpa = pos
-        else:
-            ch_names.append(items[0])
-            poss.append(pos)
+        
```

Comment:
```suggestion

```

Reply:
I am not familiar with the github operation, could you invite other developers to help me fix this change？Thanks！

Reply:
At the top of this PR page, click on "Files changed"; scroll to my first suggestion, click on "Add suggestion to batch", scroll to the next one, again click "Add suggestion to batch"; repeat until you reach the last suggestion. Then click on "Commit suggestions" and GitHub will create a new commit with my suggestions